### DISCUSSION TITLE
参考文献列表中的作者名大小写问题

### DISCUSSION THREAD

**Discussion by hmf21** [tuna/thuthesis#667] [Q&A] [answered]
使用本科生参考文献的著录格式：
\usepackage[sort]{natbib}
\bibliographystyle{thuthesis-bachelor}
之后，参考文献列表中的人名全部变成大写字母了，请问这个问题是格式所允许的嘛？如果不是，应该怎么修改回来呢

**Comment by zepinglee** [comment, marked as answer]
[10-1参考文献著录格式（2019）.docx](https://github.com/tuna/thuthesis/files/6531131/10-1.2019.docx)

**Comment by hmf21** [comment]
这里的标准参考似乎对于外文文献作者姓名大小写并未做特别规定？

**Comment by zepinglee** [comment]
GB/T 7714—2015：
<img width="718" alt="Screen Shot 2021-05-24 at 17 24 41" src="https://user-images.githubusercontent.com/12290822/119326847-ef4f0300-bcb4-11eb-9915-f6091d20f973.png">

**Comment by hmf21** [comment]
不过使用顺序编码制时，
\usepackage[sort]{natbib}
\bibliographystyle{thuthesis-numeric}
外文作者的姓名是正常出现的，请问是因为本科生的引文必须要求参照这里的字母大写的规定吗？

**Comment by zepinglee** [comment]
> 不过使用顺序编码制时，
> \usepackage[sort]{natbib}
> \bibliographystyle{thuthesis-numeric}
> 外文作者的姓名是正常出现的，请问是因为本科生的引文必须要求参照这里的字母大写的规定吗？

`thuthesis-numeric` 是按照研究生的要求。本科生往年就是按照这份“10-1参考文献著录格式（2019）.docx”的要求，具体参考系里的邮件通知。如果跟这份要求不同请及时通知我。

**Comment by hmf21** [comment]
好的感谢您～
